src/routes/console/project-[project]/auth/security/updatePersonalDataCheck.svelte

```diff
@@ -0,0 +1,66 @@
+<script lang="ts">
+    import { invalidate } from '$app/navigation';
+    import { Submit, trackError, trackEvent } from '$lib/actions/analytics';
+    import { CardGrid, Heading } from '$lib/components';
+    import { Dependencies } from '$lib/constants';
+    import { Button, Form, InputSwitch } from '$lib/elements/forms';
+    import { FormList } from '$lib/elements/forms';
+    import { addNotification } from '$lib/stores/notifications';
+    import { sdk } from '$lib/stores/sdk';
+    import { project } from '../../store';
+
+    const projectId = $project.$id;
+    let authPersonalDataCheck = $project.authPersonalDataCheck ?? false;
+
+    async function updatePersonalDataCheck() {
+        try {
+            const path = '/projects/' + $project.$id + '/auth/personal-data';
+            await sdk.forConsole.client.call(
```

Comment:
Are there any plans to add this to the SDK?

Reply:
@ArmanNik yes, Torsten said we'd do it once all the changes for 1.4 are merged.

Reply:
resolved 👍